Framework/PythonInterface/mantid/plots/mantidaxes.py

```diff
@@ -1134,6 +1134,12 @@ def __init__(self, *args, **kwargs):
         # it interfering with double-clicking on the axes.
         self.figure.canvas.mpl_disconnect(self._cids[1])
 
+    def set_title(self, *args, **kwargs):
+        # The set_title function in Axes3D also moves the title downwards for some reason so the Axes function is called
+        # instead.
+        title = super(Axes3D, self).set_title(*args, **kwargs)
```

Comment:
`super` can give unexpected results,https://www.artima.com/weblogs/viewpost.jsp?thread=281127, and I 'm not sure we can necessarily rely on the fact that `Axes3D` will always inherit from `Axes`. I'd suggest calling `Axes` directly if that's what we want: `return Axes.set_title(*args, **kwargs).`